javase/src/main/java/com/google/zxing/client/j2se/CommandLineEncoder.java

```diff
@@ -19,9 +19,12 @@
 import com.beust.jcommander.JCommander;
 import com.google.zxing.MultiFormatWriter;
 import com.google.zxing.common.BitMatrix;
+import com.google.zxing.EncodeHintType;
 
 import java.nio.file.Paths;
 import java.util.Locale;
+import java.util.Map;
+import java.util.HashMap;
```

Comment:
ok, It is not a problem, I can understand that.
I am downloading eclipse and correcting everything